### PR TITLE
fix: Added breakpoint filters backward compatibility

### PR DESCRIPTION
**Pull request by Balachandark**

### CODE REVIEW

File: inc/core/common-functions.php [hunk 1]
```diff
@@ -1267,3 +1267,65 @@ function astra_get_option_by_group( $option, $group, $default = '' ) {
...
 }
+
+if ( ! function_exists( 'astra_get_tablet_breakpoint' ) ) :
+
+	/**
+	 * Get the tablet breakpoint value.
+	 *
+	 * @param string $min min.
+	 * @param string $max max.
+	 * @return string $breakpoint.
```
Comment (premanshup) [inline inc/core/common-functions.php]: @Balachandark `@return` type should be number?

File: inc/core/common-functions.php [hunk 2]
```diff
@@ -1267,3 +1267,65 @@ function astra_get_option_by_group( $option, $group, $default = '' ) {
...
+
+		if ( '' !== $min ) {
+			$header_breakpoint = $header_breakpoint - $min;
+		} elseif ( '' !== $max ) {
+			$header_breakpoint = $header_breakpoint + $max;
+		}
+
+		/*
+		Compares if the value of header breakpoint is greater than 921
+		Usually all the actual tablet devices width in portaint mode are under 921px so if the user has added a value that is greater than 921 for header breakpoint then the default value 921 will be applied to tablet breakpoints.
```
Comment (premanshup) [inline inc/core/common-functions.php]: String `portaint` -> `portrait`

File: inc/core/common-functions.php [hunk 3]
```diff
@@ -1267,3 +1267,65 @@ function astra_get_option_by_group( $option, $group, $default = '' ) {
...
+endif;
+
+if ( ! function_exists( 'astra_get_mobile_breakpoint' ) ) :
+
+	/**
+	 * Get the mobile breakpoint value.
+	 *
+	 * @param string $min min.
+	 * @param string $max max.
+	 * @return string header_breakpoint.
```
Comment (premanshup) [inline inc/core/common-functions.php]: @Balachandark @return type should be number?

File: inc/core/common-functions.php [hunk 4]
```diff
@@ -1267,3 +1267,65 @@ function astra_get_option_by_group( $option, $group, $default = '' ) {
...
+	 */
+	function astra_get_mobile_breakpoint( $min = '', $max = '' ) {
+
+		$header_breakpoint = apply_filters( 'astra_tablet_breakpoint', 544 );
```
Comment (premanshup) [inline inc/core/common-functions.php]: Filter name is same as for Tablet, should be mobile?

`$header_breakpoint = apply_filters( 'astra_tablet_breakpoint', 544 );`

Should be:

`$header_breakpoint = apply_filters( 'astra_mobile_breakpoint', 544 );`

File: inc/core/common-functions.php (hunk 1 above)
Comment (Balachandark) [reply inc/core/common-functions.php]: I have updated the return type to number

File: inc/core/common-functions.php (hunk 3 above)
Comment (Balachandark) [reply inc/core/common-functions.php]: I have updated the return type to number

File: inc/core/common-functions.php (hunk 4 above)
Comment (Balachandark) [reply inc/core/common-functions.php]: I have updated the return filter name to astra_mobile_breakpoint

File: inc/core/common-functions.php (hunk 2 above)
Comment (Balachandark) [reply inc/core/common-functions.php]: I have corrected the spelling.

**Comment by premanshup** [comment]
Closing this PR, this has been released in the version https://github.com/brainstormforce/astra/releases/tag/v2.4.0

**Pull request closed without merging**